snap.html

```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
It's not a huge deal, but (almost) the rest of this file uses spaces and this is a tab, so maybe you could replace the tab with eight manually-typed spaces (or switch your editor to use spaces instead of tabs)?

Reply:
I do not think this is an issue because it looks alright when you view the file.

Reply:
Never mind. I just fixed this while resolving the conflict.